src/server/templates/try-it

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>{{title}}</title>
+
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="https://appsforoffice.microsoft.com/lib/1.1/hosted/excel-web-16.00.debug.js"></script>
```

Comment:
Change to excel-16-16.00.js instead of debug version.

Reply:
Actually, good catch.  You can check in as is for now, but we should make this conditional based on what the snippet specifies in its libraries, and reload the page on a mismatch (just as the regular runner does).  Please file a bug for this, adding it to the 1.2 milestone